src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -376,20 +422,31 @@ export default function FundingForm({ onFinish }: { onFinish: VoidFunction }) {
               and the resulting JBX will go to the project's owner.
             </p>
 
-            <ProjectPayoutMods
-              mods={splits.map(toMod)}
-              target={target ?? '0'}
-              currencyName={targetCurrency === ETH ? 'ETH' : 'USD'}
-              feePercentage={
-                ETHPaymentTerminalFee
-                  ? formatFee(ETHPaymentTerminalFee)
-                  : undefined
-              }
-              onModsChanged={newMods => {
-                setSplits(newMods.map(toSplit))
+            <DistributionSplitsSection
+              distributionLimit={target}
+              currencyName={V2CurrencyName(targetCurrency) ?? 'ETH'}
+              editableSplits={editingSplits}
+              lockedSplits={lockedSplits}
+              onSplitsChanged={newSplits => {
+                setEditingSplits(newSplits)
+                fundingForm.setFieldsValue({
+                  totalSplitsPercentage: getTotalSplitsPercentage(newSplits),
+                })
               }}
-              targetIsInfinite={targetType === 'infinite'}
             />
+            <Form.Item
+              name={'totalSplitsPercentage'}
+              rules={[
+                {
+                  validator: validateTotalSplitsPercentage,
+                },
+              ]}
+            >
+              {/* Added a hidden input here because Form.Item needs 
```

Comment:
**suggestion** why not a state variable for `totalSplitsPercentage`?

Or, could we compute `totalSplitsPercentage` in the validator, based on the sum of all `editingSplits` percentages?

Reply:
Yeah the field and a specific validator for it to show the error message where we want it though